src/components/gltf-model.js

```diff
@@ -0,0 +1,61 @@
+var registerComponent = require('../core/component').registerComponent;
+var THREE = require('../lib/three');
+
+var GLTFLoader = THREE.glTFLoader; // To appease the linter
+var loader = new GLTFLoader();
+var camera = null;
+var gltf = null;
+
+module.exports.Component = registerComponent('gltf-model', {
+  schema: {
+    type: 'src'
+  },
+
+  init: function () {
+    this.model = null;
+  },
+
+  update: function () {
+    var self = this;
+    var el = this.el;
+    var src = this.data;
+
+    camera = this.el.sceneEl.camera;
+    if (!src) { return; }
+
+    this.remove();
+
+    loader.load(src, function (gltfModel) {
+      var i = null;
+      var len = null;
+      var animation = null;
+
+      gltf = gltfModel;
+
+      self.model = gltfModel.scene;
+      el.setObject3D('mesh', self.model);
+      el.emit('model-loaded', {format: 'gltf', model: self.model});
+
+      if (gltf.animations && gltf.animations.length) {
+        len = gltf.animations.length;
+        for (i = 0; i < len; i++) {
+          animation = gltf.animations[i];
+          animation.loop = true;
```

Comment:
They have the object they animate nested in, but it requires expensive traversal to do that.  I'd recommend that since there's already a huge overhead for the glTF libraries not to do that but maybe add a utility to the component where you can find / cache animations to play / pause after load?